### DISCUSSION TITLE
Gitsigns not loading

### DISCUSSION THREAD

**Discussion by amandogra** [NvChad/NvChad#1511] [Q&A] [answered]
I have tried every document to figure this out in vain. No matter what I do, I see Gitsigns as not loaded.

I have not overwritten any config related to Gitsigns so my current settings is still:

```lua
-- git stuff
  ["lewis6991/gitsigns.nvim"] = {
    ft = "gitcommit",
    setup = function()
      require("core.lazy_load").gitsigns()
    end,
    config = function()
      require("plugins.configs.others").gitsigns()
    end,
  },
```

However, when I open any files in my git project, I don't see the signs. 
`:PackerStatus` command gives me following output:
```
 • gitsigns.nvim (not loaded)
```
What could I be doing wrong?

Also I have noticed `ft = "gitcommit",` in the default settings related to gitsigns. I reckon `ft` means 'filetype'. I was just wondering, is the value for it as `"gitcommit"` correct?

**Comment by siduck** [comment, marked as answer]
update nvchad

**Reply by amandogra** [reply]
Thanks. I should have checked the `issues`. Update fixed it.
